cwl/world_population_analysis.cwl

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env cwl-runner
```

Comment:
Please amend this example with respect to the new directory structure. (PR #11)

Please add also a few lines to the documentation as for the hello world example. https://github.com/reanahub/reana-demo-helloworld/pull/17/files#r169630437